test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = StreamData.fixed_map(%{points: integer(1..5)})
+      metas_map  = StreamData.fixed_map(%{
+        metas: list_of(points_map, length: 1),
+      })
+      user = nonempty(map_of(string(:printable), metas_map))
```

Comment:
This is only a minor preference, but I'd suggest using `:alphanumeric` here. It doesn't generate quite the same range of strings as `:printable` will, but the output is easier to read. I'm trading off some robustness for improved readability. If you wanted to keep `:printable`, though, I can get behind that decision 😄